core/utils.go

```diff
@@ -114,37 +114,48 @@ func ValidBumpLevel(level string) bool {
 		level == AutoLevel
 }
 
-func HighestAutoLevel(config *Config, allChanges []Change) (string, error) {
+func HighestAutoLevel(config *Config, allChanges []Change) (highestLevel string, err error) {
 	if len(allChanges) == 0 {
 		return "", ErrNoChangesFoundForAuto
 	}
 
-	highestLevel := PatchLevel
+	highestLevel = ""
+	err = ErrNoChangesFoundForAuto
 
 	for _, change := range allChanges {
 		for _, kc := range config.Kinds {
 			if kc.Label != change.Kind {
 				continue
 			}
 
-			if kc.AutoLevel == "" {
-				return "", ErrMissingAutoLevel
-			}
-
-			// major is the highest one, so we can just return it
-			if kc.AutoLevel == MajorLevel {
+			switch kc.AutoLevel {
+			case NoneLevel:
+				continue
```

Comment:
```suggestion
```
Go switch statements already don't fallthrough so we don't actually need this block at all.